USAGE_DATA.adoc

```diff
@@ -0,0 +1,28 @@
+[[odo]]
+= `odo` - Developer-focused CLI for Kubernetes and OpenShift
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+[[usage-data]]
+== Usage Data
```

Comment:
Not sure what this file's purpose is. I mean I get it. But the points under this section look half-baked. It's not clear that x, y and z info will be collected when the preference is set to True. 

Are we collecting the OS type and odo version in use event= when preference is set to `False`?

Reply:
>Are we collecting the OS type and odo version in use event= when preference is set to False?

```
* when `ConsentTelemetry` preference is set to `false`
* system information such as OS type
* `odo` version in use
```
All of the above are different events but I can see why it is confusing. If the user had set the preference to true earlier and then changed it to false, the setting false action will be sent to Segment. The last 2 points about OS type and odo version are unrelated to the setting preference to false point.

I'll work on fully baking it!

Reply:
Yeah, it's confusing. We could just document what we're doing if the telemetry is enabled. Let's make it simple to understand.